src/main/groovy/edu/oregonstate/mist/locations/frontend/mapper/LocationMapper.groovy

```diff
@@ -50,5 +52,28 @@ class LocationMapper {
 
         // hashCode is used as metadata in ES. No need to expose it
         ro?.attributes?.remove("hashCode")
+
+        // expose isOpen field
+        def now = DateTime.now(DateTimeZone.UTC)
```

Comment:
this function would be easier to test if `now` were supplied as an argument

Reply:
nevermind. thanks for pointing out the existence of the DateTimeUtils.setCurrentMillisFixed function :)